lua/caramba/git.lua

```
@@ -519,30 +524,181 @@ M.review_code = function()
+  end
+  
+  if #found_configs > 0 then
+    conventions = "\n## Project Configuration:\n"
+    for _, config in ipairs(found_configs) do
+      conventions = conventions .. "- " .. config .. " detected\n"
+    end
+  end
+  
+  -- Get git context if available
+  local git_context = ""
+  local git_status = vim.fn.system("git status --porcelain " .. vim.fn.expand('%:p'))
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)